site/tsconfig.test.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    "target": "es6",
```

Comment:
Needs `es6` otherwise the compiler breaks stuff. See for more details https://slate-js.slack.com/archives/C1RH7AXSS/p1647905084771509